modules/ROOT/pages/vpc-tutorial.adoc

```diff
@@ -51,7 +51,7 @@ If you don't select a business group, the Anypoint VPC is associated with the ma
 image::create-vpc.png[]
 +
 . Click *Firewall Rules* to expand the fields and xref:vpc-firewall-rules-concept.adoc[configure firewall rules]. +
-By default, all inbound traffic is blocked, and you need to configure firewall rules to allow traffic to your worker. You can configure these rules at a later time. +
+By default, all inbound traffic is open, and you need to configure firewall rules to block external traffic to your worker. You can configure these rules at a later time. +
```

Comment:
Not true. By default traffic is allowed from all sources to the public listener ports; 8081 for HTTP and 8082 for HTTPS. Please DON'T put all inbound traffic.